core/src/test/java/com/taobao/arthas/core/util/StringUtilsTest.java

```diff
@@ -18,4 +27,375 @@ public void testHumanReadableByteCount() {
         Assert.assertEquals(StringUtils.humanReadableByteCount(1024L * 1024 * 1024 * 1024L), "1.00 TiB");
         Assert.assertEquals(StringUtils.humanReadableByteCount(1024L * 1024 * 1024 * 1024 * 1024), "1.00 PiB");
     }
-}
\ No newline at end of file
+
+    @Test
+    public void testCause() {
+        Assert.assertNull(StringUtils.cause(new Throwable(null, null)));
+
+        final Throwable t2 = new Throwable("error message", new Throwable("error message", null));
+        Assert.assertEquals(t2.getMessage(), StringUtils.cause(t2));
+    }
+
+    @Test
+    public void testCommaDelimitedListToSet() {
+        TreeSet set = new TreeSet();
```

Comment:
Couldn't you have done: `TreeSet set = new TreeSet(){"foo"}`?

Reply:
apparently not